infra-as-code/bicep/machinelearning.bicep

```diff
@@ -189,8 +183,6 @@ resource aiHub 'Microsoft.MachineLearningServices/workspaces@2024-07-01-preview'
       authType: 'AAD'
       category: 'AzureOpenAI'
       isSharedToAll: true
-      useWorkspaceManagedIdentity: true

```

Comment:
Without `useWorkspaceManagedIdentity: true` how does the workspace's managed identity get selected to be used?  The default for this is `false`, it was intentional to set this to `true` to be explicit.

Reply:
still present